language/move-lang/tests/move_check/typing/module_call_visibility_friend_invalid.exp

```diff
@@ -0,0 +1,11 @@
+error: 
+
+   ┌── tests/move_check/typing/module_call_visibility_friend_invalid.move:5:20 ───
+   │
+ 5 │     public(friend) f_friend() {}
+   │                    ^^^^^^^^ Unexpected 'f_friend'
+   ·
+ 5 │     public(friend) f_friend() {}
+   │                    -------- Expected 'fun'
```

Comment:
I don't think this is the set of errors you expected...

Reply:
My bad. I am missing the `fun`...

Reply:
Fixed in ea3a8c4